### PR TITLE
Adding helper functions to fix code smells and comments

### PR DESCRIPTION
**Pull request by julianmarrero**
Created helper functions to reduce repeated code. Added comments to methods to make it clear functionality of method to address feedback from previous pull requests.

### CODE REVIEW

**Review by JerryLi-617 — APPROVED** [review]
Great use of helper methods, everything looks clear

File: app/src/main/java/com/example/notes2text/adapters/FileEditingAdapters/FileEditorActivity.java [hunk 1]
```diff
@@ -51,18 +56,29 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
         Bundle bundle = getIntent().getExtras();
-        // Receiving file from ActivitySwitchController
-        file = (File)bundle.getSerializable("file");
-        // Receiving array of selected files from JoinController
-        ArrayList<File> selectedFiles = (ArrayList<File>)bundle.getSerializable("selectedFiles");
-
-        // if received file from ActivitySwitchController
-        if (file != null) {
+        // Receiving file from ActivitySwitchController or SavePopUpActivity
```
Comment (ChowdhuryShaswata) [inline app/src/main/java/com/example/notes2text/adapters/FileEditingAdapters/FileEditorActivity.java:59]: Thanks for adding in the explanations as per my last request - it is all clear now.

**Review by ChowdhuryShaswata — APPROVED** [review]

**Comment by Thishor** [comment]
Good comments. Easy to understand code

**Review by Thishor — APPROVED** [review]
Good documentation throughout. Easy to follow code.

**Pull request merged**